tests/ui/test_quickstarts.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.framework.testlib import ui, skipif_ocs_version, tier2
+
+logger = logging.getLogger(__name__)
+
+
+class TestPvcUserInterface(object):
+    """
+    Test Quickstarts in the User Interface
+
+    """
+
+    @ui
+    @tier2
+    @skipif_ocs_version("<4.7")
+    @pytest.mark.parametrize(
+        argnames=["title"],
+        argvalues=[
+            pytest.param(
+                *["Getting started with OpenShift Container Storage"],
+                marks=pytest.mark.polarion_id("OCS-2458"),
+            ),
+            pytest.param(
+                *["OpenShift Container Storage Configuration & Management"],
+                marks=pytest.mark.polarion_id("OCS-2466"),
+            ),
+        ],
+    )
+    def test_quickstart_presense(self, setup_ui, title):
+        """
+        Check that the quickstart with the given title is present in the UI
+
+        """
+        quickstart_ui_obj = PageNavigator(setup_ui)
+        quickstart_ui_obj.navigate_quickstarts_page()
+        quickstart_ui_obj.take_screenshot()
```

Comment:
take_screenshot is enabled by default

Reply:
By default the screenshot is taken before the click, and I needed an extra screenshot after the click.